### PR TITLE
Add release actions

### PR DESCRIPTION
**Pull request by MartinHjelmare**
- check-versions will compare the python version in `python/imjoy_rpc/VERSION` and the javascript version in `javascript/package.json` and exit with 1 if they are not equal.
- release will also check versions before running the Python build and checking for a version bump. If a version bump is found, it will release to PyPI and then run the Javascript build and release to npm.
- release-drafter creates draft releases with auto-labeling of PRs and automatic headers taken from labels.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      - name: Build package
+        run: |
+          python setup.py sdist bdist_wheel
+
+      - name: Publish package on PyPI
+        if: steps.check-version.outputs.tag
+        uses: pypa/gh-action-pypi-publish@v1.4.2
+        with:
+          user: __token__
+          password: "${{ secrets.PYPI_TOKEN }}"
```
Comment (MartinHjelmare) [inline .github/workflows/release.yml:84]: We need to configure a secret `PYPI_TOKEN`.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        with:
+          node-version: 15.5.0
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
+      - run: npm run check-format
+      - run: npm run test
+      - run: npm run build
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (MartinHjelmare) [inline .github/workflows/release.yml:110]: And a secret `NPM_TOKEN`.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+jobs:
+  check:
+    name: Check versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 2
```
Comment (MartinHjelmare) [inline .github/workflows/release.yml:17]:
```suggestion
```

**Comment by MartinHjelmare** [comment]
The draft-release workflow will not pass until this PR is merged.

File: .github/workflows/release.yml (hunk 1 above)
Comment (oeway) [reply .github/workflows/release.yml:82]: Done

File: .github/workflows/release.yml (hunk 2 above)
Comment (oeway) [reply .github/workflows/release.yml:108]: Done

**Comment by oeway** [comment]
@MartinHjelmare I have merged it but it seems there are some issue still with the requirements file for the release actions: https://github.com/imjoy-team/imjoy-rpc/runs/2430145612#step:7:58

**Comment by MartinHjelmare** [comment]
We have to cd to the correct directory in the same step as the install command since steps are run in separate processes.

**Pull request merged**